django_admin_keyboard_shortcuts/static/admin/js/shortcuts.js

```diff
@@ -11,7 +11,17 @@ function isFocusedTextField() {
 {
     let previousKey = undefined;
     const shortcutFunctions = new Map([
-        ["g i", () => { document.location.href = "/admin/"; }],
+        ["g i", () => {
+            // Get the admin index URL from the json_script element
+            const adminIndexElement = document.getElementById('admin-index-url');
+            if (adminIndexElement) {
+                const adminIndexUrl = JSON.parse(adminIndexElement.textContent);
+                document.location.href = adminIndexUrl;
+            } else {
+                // Fallback to default if element not found
+                document.location.href = "/admin/";
+            }
```

Comment:
I don't really see a point to the conditional, it will always be true.

I also don't know if it's needed to use `JSON.parse()` or not. Maybe it is, but I feel like it's just a string so it isn't really doing anything.